openreview/openreview.py

```diff
@@ -1084,21 +1084,26 @@ def add_members_to_group(self, group, members):
         """
         Adds members to a group
 
-        :param group: Group to which the members will be added
-        :type group: Group
+        :param group: Group (or Group's id) to which the members will be added
+        :type group: Group or str
         :param members: Members that will be added to the group. Members should be in a string, unicode or a list format
         :type members: str, list, unicode
 
         :return: Group with the members added
         :rtype: Group
         """
         def add_member(group, members):
+            group_id = ''
```

Comment:
`group_id = [group if type(group) is string_types else group.id]`

Reply:
Done